### PR TITLE
Add our names to README.md

### PR DESCRIPTION
**Pull request by Nicafe**
I added our names to README.md file. Hope everybody has a nice weekend! ❤️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# NCFH
\ No newline at end of file
+# NCFH
+* I'm Nicolas
+* My wife's Cassie
```
Comment (F-guy) [inline README.md]: it should be "my wife is cassie"

Comment (F-guy) [inline README.md]: same for the other sentences

**Comment by F-guy** [comment]
make the changes I said and then request again 😄

**Comment by Nicafe** [comment]
OK. 😄

**Pull request merged**